bw_matchbox/assets/templates/common-modal.html

```diff
@@ -1,3 +1,12 @@
+{#
+
+NOTE: You need to include nexde if you want to use common modal feature on the page:
```

Comment:
Can you fix the typo? I am not sure what the intent is here.

Reply:
What do you mean? 'Nexde'? It was for 'next code' -- smb has stolen my letters.

Anyway, I'm moving this short howto into the `common-modal.md`. That's better.

See 83e8d6a in dev-230808-56-match-highlight-items